### DISCUSSION TITLE
Is "Indexer API Reference page" up to date?

### DISCUSSION THREAD

**Discussion by wenceslas-sanchez** [aptos-labs/aptos-developer-discussions#479] [Dev Doc and Tutorial Feedback]
Hi team, do you know why in the Indexer API Reference page on the Aptos documentation (https://aptos.dev/en/build/indexer/indexer-reference) there are missing table listed in Hasura example (for instance the table current_delegated_staking_pool_balances is present in Hasura but not in the documentation); is the official documentation not up to date?

**Comment by ying-w** [comment]
The indexer-reference page hasn't been [updated in a while](https://github.com/aptos-labs/developer-docs/commits/main/apps/nextra/pages/en/build/indexer/indexer-reference.mdx), you can find additional information about the tables in the [processor repo](https://github.com/aptos-labs/aptos-indexer-processors), for that specific table, the logic is

https://github.com/aptos-labs/aptos-indexer-processors/blob/4cc90a3022b4736ce6f8c02fd55f65c13ed0bb1e/rust/processor/src/processors/stake_processor.rs#L332

**Reply by wenceslas-sanchez** [reply]
Understood thank you!
